src/js/view/controls/components/settings/submenu.js

```diff
@@ -12,6 +12,16 @@ export default function SettingsSubmenu(name, categoryButton, isDefault) {
     categoryButtonElement.className += ' jw-submenu-' + name;
     categoryButton.show();
 
+    const nextSubmenu = function(evt) {
```

Comment:
This function name is confusing. Is this a function go to the next submenu? Each submenu should not know about other submenus (and currently do not).

Reply:
I'll rename it. What this does is that when you click tab after the last submenu item, the focus goes back to the submenu's category element. then the default tab behaviour kicks in, which would move the focus to the next item in the topbar.